.pre-commit-config.yaml

```diff
@@ -4,5 +4,5 @@ repos:
     hooks:
       - id: black
         exclude: ^mesa/cookiecutter-mesa/
-        language_version: python3.6
+        language_version: python3
```

Comment:
After testing this, I can confirm that you can delete this line altogether. There is no need for it.